TheMET/TheMET/DataModels/Department.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-class Department {
+class Department: Decodable {
     
     let departmentId: Int
```

Comment:
Please rename `departmentId` to `id` as the class name already provides a context of this name. Don't forget to fix the CodingKeys

Reply:
fixed